os/arch/arm/src/armv8-m/up_exception.S

```diff
@@ -420,4 +451,13 @@ g_nestlevel:
 	.size   g_nestlevel, 4
 #endif
 
+	.bss
+	.global	g_bakirqstkalloc
```

Comment:
This is my guess, `bak` means backup?
How about renaming it as g_nestedintstk?

Reply:
Done